### PR TITLE
Use exact image versions in docker-compose

### PR DESCRIPTION
**Pull request by ashishnitinpatil**
Using [:latest is quite bad](https://vsupalov.com/docker-latest-tag/), since docker does not keep .lock files like Pipfile. When I cloned the repo & ran `docker-compose up --build`, my docker used redis image v3 whereas latest is actually v5.

**Note -** After running `py.test` in the docker container (`web`), I am getting a single error (1393 passed, 2 warnings, 1 error) `ERROR at setup of test_email_sending_asynchronously` (`site-packages/celery/backends/base.py:508: TimeoutError`).

### Pull Request Checklist

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [ ] Database migration files are up to date.
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] GraphQL schema and type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ services:
       - redis
 
   db:
-    image: library/postgres:latest
+    image: library/postgres:11.2
```
Comment (jxltom) [inline docker-compose.yml]: In the tests we are using ```9.4``` https://github.com/mirumee/saleor/blob/1a2480fa0c08a6730bf00fca536b68c52f10a7ba/.circleci/config.yml#L28, maybe we could keep them same?

Comment (ashishnitinpatil) [reply docker-compose.yml]: Thanks, new to saleor, so didn't notice the Circle CI config also has docker images. Will check if similar thing is there for redis image & will update the PR accordingly.

Comment (NyanKiyoshi) [reply docker-compose.yml]: The latest is fine as well though.

**Comment by ashishnitinpatil** [comment]
I switched back Redis image version to 4.0 as well because I suspected v5.0 may not yet be supported by python libraries we depend on (`django-redis`, `redis`, etc.). e.g. https://github.com/niwinz/django-redis/issues/364 

I ran the `py.test` again, will report after it's done (~15 mins).

**Comment by ashishnitinpatil** [comment]
Alright, so both `postgres:9.4-alpine` & `redis:4.0-alpine` fail during the build itself with the following errors -

```
redis_1   | 1:M 18 Feb 13:37:31.142 # Can't handle RDB format version 9
redis_1   | 1:M 18 Feb 13:37:31.142 # Fatal error loading the DB: Invalid argument. Exiting.
db_1      | LOG:  unrecognized configuration parameter "max_wal_size" in file "/var/lib/postgresql/data/postgresql.conf" line 213
db_1      | LOG:  unrecognized configuration parameter "min_wal_size" in file "/var/lib/postgresql/data/postgresql.conf" line 214
db_1      | FATAL:  configuration file "/var/lib/postgresql/data/postgresql.conf" contains errors
```

I switched to `redis:5.0-alpine` & `postgres:9.6-alpine` & redis built successfully, but postgres failed with 
```
db_1      | FATAL:  "/var/lib/postgresql/data" is not a valid data directory
db_1      | DETAIL:  File "/var/lib/postgresql/data/PG_VERSION" does not contain valid data.
db_1      | HINT:  You might need to initdb.
```
Not going to bother with postgres-10 because [psycopg2 v2.7.6.1](https://github.com/mirumee/saleor/blob/master/Pipfile.lock#L719) supports [postgres 11.1](http://initd.org/psycopg/docs/news.html#what-s-new-in-psycopg-2-7-6-1), so will settle with that for now. Updating PR once I rerun `py.test`.

**Comment by jxltom** [comment]
If postgres9.4 failed in docker-compose, why the tests passes in circle CI?

**Comment by ashishnitinpatil** [comment]
@jxltom I'm not sure. But the docker build itself failed for me, probably becaues Circle CI build is limited in nature as it's config suggests, compared to our postgres container.

**Comment by ashishnitinpatil** [comment]
Restored original redis version (5.0) & changed postgres to 11.1 given `pyscopg2` support. `py.test` threw the same single error mentioned in PR description. Seems good to go to me at this moment (except for the circle CI version mismatch).

**Comment by maarcingebala** [comment]
@ashishnitinpatil Thanks for opening this. Would you mind updating the changelog?

**Comment by ashishnitinpatil** [comment]
Done @maarcingebala :)

**Comment by maarcingebala** [comment]
Great! Congrats on your first PR to Saleor!

**Comment by ashishnitinpatil** [comment]
Aye Sir! Happy to be "on-board" :D

**Comment by chetabahana** [comment]
I am pulling the current version of Postgres 11.2-1 and Redis 5.0.4. Seems both are find.
```
"Env": [
    "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
    "GOSU_VERSION=1.10",
    "REDIS_VERSION=5.0.4",
    "REDIS_DOWNLOAD_URL=http://download.redis.io/releases/redis-5.0.4.tar.gz",
    "REDIS_DOWNLOAD_SHA=3ce9ceff5a23f60913e1573f6dfcd4aa53b42d4a2789e28fa53ec2bd28c987dd"
],

"Env": [
    "POSTGRES_USER=saleor",
    "POSTGRES_PASSWORD=saleor",
    "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/lib/postgresql/11/bin",
    "GOSU_VERSION=1.11",
    "LANG=en_US.utf8",
    "PG_MAJOR=11",
    "PG_VERSION=11.2-1.pgdg90+1",
    "PGDATA=/var/lib/postgresql/data"
],
```

**Pull request merged**